livre/lib/configuration/ApplicationParameters.cpp

```diff
@@ -49,7 +49,7 @@ const std::string TRANSFERFUNCTION_PARAM = "transfer-function";
 
 ApplicationParameters::ApplicationParameters()
     : Parameters( "Application Parameters" )
-    , cameraPosition( 0, 0, 1 )
```

Comment:
Why modified the default value ? Please  do not modify default values unless you have a reason.